packages/account/src/Sections/Verification/ProofOfOwnership/Card.jsx

```diff
@@ -58,6 +68,7 @@ const Card = ({ card, handleChange, handleBlur, values, setFieldValue, index, er
                     index={index}
                     error={error}
                     validateField={validateField}
+                    disableSubmitButton={disableSubmitButton}
```

Comment:
i think instead of drill down this prop we can use formik for form validation 
it is more clear and cleaner

Reply:
Hello @ali-hosseini-fs,

That function is for when the x icon is clicked. The x icon doesn't trigger form events, that is why I used the function